spec/scss_lint/linter/url_format_spec.rb

```diff
@@ -31,6 +31,12 @@
 
       it { should_not report_lint }
     end
+
+    context 'when URL contains a variable' do
+      let(:url) { '${' }
```

Comment:
@trotzig Because in some of our cases we use variables because the CSS goes into a CMS that replaces those variables but while we are prototyping it never passes linter because of that